usaspending_api/disaster/v2/views/disaster_base.py

```diff
@@ -49,6 +51,10 @@ def filters(self):
     def def_codes(self):
         return self.filters["def_codes"]
 
+    @property
+    def reporting_period_max(self):
+        return datetime.now().date()
```

Comment:
If max date is always "today" is it necessary?

Reply:
Wouldn't we want max date to make sure that we only consider submissions that are not part of an active submission window?

Reply:
Yes, but that date will come from the future `dabs_submission_window_schedule` table